### DISCUSSION TITLE
Examples

### DISCUSSION THREAD

**Discussion by lalzart** [uicrooks/shopify-theme-lab#41] [General]
I see some really interesting things in the template but it would be nice to see them implemented a couple more times to better understand.

**Comment by sergejcodes** [comment]
Hi lalzart 👋

this repo is kept barebones intentionally.

There is, however, the [Shopify Foundation Theme repo](https://github.com/uicrooks/shopify-foundation-theme) which is built with Shopify Theme Lab. There you can find some real-life examples.
